store/evm_aux/store.go

```diff
@@ -88,6 +94,17 @@ func (s *EvmAuxStore) SetBloomFilter(tran *leveldb.Transaction, filter []byte, h
 	return tran.Put(bloomFilterKey(height), filter, nil)
 }
 
+func (s *EvmAuxStore) IsDupEVMTxHash(txHash []byte) bool {
```

Comment:
Didn't we agree to load the whole lot once on startup?

Reply:
Yes fixed